src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,23 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in address book by name
+     * @return success/failure of operation
+     */
+    public boolean sort() {
+        if(internalList.isEmpty()){
+            return false;
+        }
+        Collections.sort(internalList, new Comparator<Person>() {
```

Comment:
Maybe you can declare a PersonComparator class for a more organized and abstracted code?